bin/runtime-common/src/messages.rs

```diff
@@ -134,7 +115,13 @@ impl<
 	}
 }
 
-/// This chain that has `pallet-bridge-messages` and `dispatch` modules.
+/// Chain that has `pallet-bridge-messages` module.
+pub trait ChainWithMessages {
```

Comment:
@svyatonik 
I started to work on the same thing yesterday :D - to simplify this ChainWithMessages, because the name is not very clear and also we have two different traits with the same name `trait ChainWithMessages` and it causes some unnecessery small duplications

https://github.com/paritytech/parity-bridges-common/compare/bko-nits?expand=1

but I am reverting it as you work on the same :)

Reply:
Sorry for that :)